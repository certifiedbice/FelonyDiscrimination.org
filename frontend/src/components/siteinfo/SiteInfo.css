#site-info {
    display: flex;
    flex-direction: row;
    justify-content: center;
    margin: 5px 0 0 0;
}

#site-info-logo {
    margin-left:-38px;
    flex: 1;
    background-image: url(../../images/site-logo.png);
    background-repeat: no-repeat;
    background-size: contain;
    background-position: center;
    cursor: pointer;
}

#site-info-title {
    flex: 2;
    margin-top: 5px;
    margin-left: -53px;
}

#site-info-title h1 {
    margin: 0;
    font-size: 16px;
    font-weight: 700;
}

.site-info-link {
    text-decoration: none;
    color: #e6e7e8;
}

#site-info-title a:hover {
    color: #4378bc;
}

#site-info-title a:active {
    color: #4378bc;
}

/* Not working for some reason, causes hover to not change */
/* #site-info-title a:visited {color: #e6e7e8;} */

/* Desktop CSS */
@media(min-width:991px){
    
    #site-info {
        width: 50%;
        margin: 10px auto 0 auto;
    }
    
    #site-info-title {
        margin-top: 3px;
        margin-left: -20px;
    }

    #site-info-logo {
        margin-left:-40px;
    }

    #site-info-title {
        margin: 30px 0 0 -60px;
    }

    #site-info-title h1 {
        font-size: 35px;
    }

}