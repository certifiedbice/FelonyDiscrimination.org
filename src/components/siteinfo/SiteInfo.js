import React from 'react';
import {Link} from 'react-router-dom';
import './SiteInfo.css';
export default function Header(){
	return(
		<div id="site-info">
				<div id="site-info-logo" aria-label="FelonyDiscrimination.org logo">
					<Link className='site-info-link' to='/'></Link>
				</div>
				<div id="site-info-title">
					<Link className='site-info-link' to='/'><h1>FelonyDiscrimination.org</h1></Link>
				</div>
			</div>
	);
}
