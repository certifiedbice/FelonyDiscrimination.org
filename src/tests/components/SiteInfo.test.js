import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import SiteInfo from '../../components/siteinfo/SiteInfo';

it('renders without crashing',()=>{
    const div=document.createElement('div');
    ReactDOM.render(<BrowserRouter><SiteInfo/></BrowserRouter>,div);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<BrowserRouter><SiteInfo/></BrowserRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});