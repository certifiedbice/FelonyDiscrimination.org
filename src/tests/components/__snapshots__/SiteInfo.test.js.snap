// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="site-info"
>
  <div
    aria-label="FelonyDiscrimination.org logo"
    id="site-info-logo"
  >
    <a
      className="site-info-link"
      href="/"
      onClick={[Function]}
    />
  </div>
  <div
    id="site-info-title"
  >
    <a
      className="site-info-link"
      href="/"
      onClick={[Function]}
    >
      <h1>
        FelonyDiscrimination.org
      </h1>
    </a>
  </div>
</div>
`;
